engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -6265,13 +6274,15 @@ void demon_hunter_t::create_cooldowns()
   cooldown.movement_shared      = get_cooldown( "movement_shared" );
 
   // Vengeance
-  cooldown.demon_spikes         = get_cooldown( "demon_spikes" );
-  cooldown.fiery_brand          = get_cooldown( "fiery_brand" );
-  cooldown.sigil_of_chains      = get_cooldown( "sigil_of_chains" );
-  cooldown.sigil_of_flame       = get_cooldown( "sigil_of_flame" );
-  cooldown.sigil_of_misery      = get_cooldown( "sigil_of_misery" );
-  cooldown.sigil_of_silence     = get_cooldown( "sigil_of_silence" );
-  cooldown.fel_devastation      = get_cooldown( "fel_devastation" );
+  cooldown.demon_spikes                      = get_cooldown( "demon_spikes" );
+  cooldown.fiery_brand                       = get_cooldown( "fiery_brand" );
+  cooldown.sigil_of_chains                   = get_cooldown( "sigil_of_chains" );
+  cooldown.sigil_of_flame                    = get_cooldown( "sigil_of_flame" );
+  cooldown.sigil_of_misery                   = get_cooldown( "sigil_of_misery" );
+  cooldown.sigil_of_silence                  = get_cooldown( "sigil_of_silence" );
+  cooldown.fel_devastation                   = get_cooldown( "fel_devastation" );
+  cooldown.volatile_flameblood_icd           = get_cooldown( "volatile_flameblood_icd" );
+  cooldown.volatile_flameblood_icd->duration = 1_s;
```

Comment:
It might be technically better to grab this duration from the talent.vengeance.volatile_flameblood internal cooldown in the spell data.

Reply:
`talent.vengeance.volatile_flameblood->internal_cooldown()` evaluates to 0 from my testing of it.

Reply:
```
Name             : Volatile Flameblood (id=390808) [Spell Family (107), Passive] 
Talent Entry     : Vengeance [tree=spec, row=6, col=4, max_rank=1, req_points=8]
Class            : Demon Hunter
School           : Physical
Spell Type       : None
Internal Cooldown: 0.9 seconds
```

This is from the spell data which comes from  spell->internal_cooldown().total_seconds()